subjects/ai/training/audit/README.md

```diff
@@ -16,29 +16,19 @@
 
 #### Exercise 1: MSE Scikit-learn
 
-##### The goal of this exercise is to learn to use `sklearn.metrics` to compute the mean squared error (MSE).
+###### Is the Mean Squared Error (MSE) calculated using `sklearn.metrics` library?
```

Comment:
Article missing.
Is the Mean Squared Error (MSE) calculated using **THE** `sklearn.metrics` library?